examples/with_winit/src/main.rs

```diff
@@ -49,10 +53,51 @@ struct Args {
     /// Switch between scenes with left and right arrow keys
     #[arg(long)]
     scene: Option<i32>,
+    #[arg(
```

Comment:
This is missing some docs

It can probably also reasonably move to `scenes`. It seems equally applicable to `headless`

Reply:
Good idea! Done.